.swiftformat

```diff
@@ -1,4 +1,108 @@
-# https://github.com/nicklockwood/SwiftFormat/blob/main/Rules.md
+# SwiftFormat config compliant with Google Swift Guideline
+# https://google.github.io/swift/#control-flow-statements
 
-# file options
---exclude Pods
\ No newline at end of file
+# Specify version used in a project
+
+--swiftversion 5.7
```

Comment:
Can we use swift version 5.9 ?

Reply:
I have added a `.swift-version` file  to specify swift version in 41bcc869 , if use `--swiftversion`, can we remove that file?

Reply:
This file also contains other rules which is consistent with swiftlint configuration. You may edit rules in this file but we'd better not to remove it.

Reply:
ok